main.py

```diff
@@ -286,6 +319,44 @@ def validateSettings(staticLetters) -> bool:
     return True
 
 
+def normalized(sett: dict) -> list:
+    total = sum(sett[i] for i in sett)
+    return [(sett[i]/total, i) for i in sett]
+
+
+def orderLetters(configs: tuple = BIGRAMS_CONFIGS) -> str:
```

Comment:
Please choose a name…
- which implies that things get loaded
- which doesn't imply that some letters should be the argument for this function

Putting "monograms" somewhere in the name (and removing "letters") might be a good idea.

Reply:
Done in 444e09cfcf29a2dc6c1ad61332d4f2e98f975ae6